README.md

```diff
@@ -50,6 +50,39 @@ For more advanced configuration, please refer to the `curl` documentation.
 
 ## Development
 
+### System Requirements
+Install the following with your system's package manager
+ - opam
+ - make
+ - clang
+ - patch
+ - curl
+ - libcurl-devel
```

Comment:
Are these necessary to install opam? Or for installing packages with opam?

Reply:
Patch, clang and make were prereqs for opam

Curl and libcurl-devel were requirements for ocurl, which itself was a requirement to build the tool with dune.